connectors/connectors_cli.py

```diff
@@ -147,6 +149,16 @@ def validate_language(ctx, param, value):
     return value
 
 
+# override click's default 'choices' prompt with something a bit nicer
+def interactive_service_type_prompt():
+    options = list(_default_config()["sources"].keys())
+    print(f"{Fore.GREEN}?{Style.RESET_ALL} Service type:")  # noqa: T201
+    result = TerminalMenu(
+        options, menu_cursor_style=("fg_green",), clear_menu_on_exit=False
```

Comment:
I would add these options: 

```
show_search_hint=True
search_key=None
```
To let customers know that they can type to search.